### PR TITLE
Set up tests

### PR DESCRIPTION
**Pull request by Fantaskink**
- Added route test
- Added utils tests
- Refactored microservice

### CODE REVIEW

**Review by TheMOHD3886 — DISMISSED** [review]
lgtm bruva 👍

File: test/utils/convertBigIntToString.test.js
Comment (MortenMunk) [inline test/utils/convertBigIntToString.test.js:1]: We should probably test edge cases where ytou try to convert a char or string to bigint, so we know if behaves how we want (e.g throws an error)

File: test/utils/stopContract.test.js
Comment (Mafusn) [inline test/utils/stopContract.test.js:1]: Put into one Describe() only

File: test/utils/startContract.test.js
Comment (Mafusn) [inline test/utils/startContract.test.js:1]: Put into one Describe() only

File: test/routes/partyRoutes.test.js
Comment (Mafusn) [inline test/routes/partyRoutes.test.js:1]: Put into one Describe() only i think, not so sure about this one

File: test/routes/candidateRoutes.test.js
Comment (Mafusn) [inline test/routes/candidateRoutes.test.js:1]: Put into one Describe() only

File: utils/startContract.js
Comment (MortenMunk) [inline utils/startContract.js:1]: Remember to have linter ignore console logs, if they are meant to be in the code

**Review by MortenMunk — CHANGES_REQUESTED** [review]
Just some consideration for testing, and some linting reminders

**Review by Mafusn — CHANGES_REQUESTED** [review]
LGTM, maybe too few test cases

**Review by Mafusn — DISMISSED** [review]
Lets goooo

**Review by MortenMunk — DISMISSED** [review]
LGTM once linting is fixed

**Review by TheMOHD3886 — APPROVED** [review]
lgtm dawg 👍

**Review by Mafusn — APPROVED** [review]

**Pull request merged**